### PR TITLE
Refactor callback query handling

### PR DESCRIPTION
**Pull request by YSRoot**
Writing methods for each callback in the handler takes up a lot of space in the class, how about writing your own handler for each callback? 
Before finishing the callback rework, I wanted to share a draft. does anyone have any thoughts or recommendations?

I suggest:
1) for each callback make separate Callback/~~CallbackData~~ classes
2) store a list of all available callbacks for the bot in the config

~~and small change for callback_data string formatting `action?param1=value1&param2=value2&...`~~

Sample:
1) add callback in config
```
    'bots' => [
        'myFirstBot' => [
            'callbacks' => [
                App\Telegraph\Callbacks\OkCallback::class,
            ],
        ],
    ],
```

2) create callback handle class
```
<?php

namespace App\Telegraph\Callbacks;

final class OkCallback extends Callback
{
    public static string $name = 'sample-action';

    public function handle(): void
    {
//        $this->bot
//        $this->chat
//        $this->request
//        $this->callbackQuery
//        $this->messageId
//        $this->callbackQueryId
//        ...
    }
}
```

### CODE REVIEW

**Comment by YSRoot** [comment]
@fabio-ivona what do you think?

**Comment by fabio-ivona** [comment]
@YSRoot interesting idea

Will have a look at it

File: src/Keyboard/Button.php [hunk 1]
```diff
@@ -12,17 +14,17 @@ class Button
 
     private int $width = 0;
 
-    private function __construct(
+    final public function __construct(
```
Comment (fabio-ivona) [inline src/Keyboard/Button.php:17]: may I ask why are you changing this one's visibility?

File: config/telegraph.php [hunk 2]
```diff
@@ -63,4 +72,12 @@
         'bot' => DefStudio\Telegraph\Models\TelegraphBot::class,
         'chat' => DefStudio\Telegraph\Models\TelegraphChat::class,
     ],
+
+    'bots' => [
+        'myFirstBot' => [
```
[the remark this answers is not in the document]
Comment (YSRoot) [reply config/telegraph.php:77]: Store them in a database?

For example, add a json field `settings` to the telegraph_bots table (or create a separate table `telegraph_actions:id,type (command/callback),action,class` if you think that there may be too many actions for a column in the table). In which we will store the name of callbacks and commands that can be executed by the bot.

For automate the addition or removal of commands/callbacks, make an artisan command that will index certain directories and get classes from there that implement interfaces/abstractions of commands or callbacks. And will they be added to the selected bot or to all?

And add methods for adding and removing a command/callback from the bot.

File: src/Keyboard/Button.php (hunk 1 above)
Comment (YSRoot) [reply src/Keyboard/Button.php:17]: I don't know, maybe I was drunk

File: config/telegraph.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (fabio-ivona) [reply config/telegraph.php:77]: I'll have think about it, there's should be a cleaner way

**Comment by fabio-ivona** [comment]
I'm closing this because it seems inactive, please let me know if you are willing to keep working and I'll open it again

**Pull request closed without merging**